aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -92,6 +92,9 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
     // Using SHA2-256's cost
     [.hash.ripemd160.base, optional "hash.ripemd160.base", 3000],
     [.hash.ripemd160.per_byte, optional "hash.ripemd160.per_byte", 50],
+    // Using SHA2-256's cost
+    [.hash.blake2b_256.base, optional "hash.blake2b_256.base", 3000],
```

Comment:
Can you rebase and change to gas schedule version here instead of optional?

Reply:
how do I do that change?

Reply:
Actually https://github.com/aptos-labs/aptos-core/pull/5794 has not been merged yet. It's fine to leave this be then. cc @vgao1996 for rebasing later and updating this new entry to use gas schedule version as well.

Reply:
Fixed.